lib/tasks/create_library_types.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```

Comment:
Again maybe worth adding a test. This came out of Seena setting up service as all of the tasks were broken. Also worth adding it to readme.

Reply:
Again also worth adding some feedback once the task has completed successfully.